superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -170,25 +182,91 @@ const dropdownIconsStyles = css`
   }
 `;
 
+export const handleDropdownNavigation = (
```

Comment:
I know that this PR is focused on dashboard interactions, but maybe (future iterations) we should add this functionality to the main Dropdown component, so that the keyboard navigation works in all dropdowns?

Reply:
Yeah, good idea!